README.md

```diff
@@ -9,247 +9,128 @@ Also, see the [API documentation](https://aws.github.io/aws-encryption-sdk-c/htm
 
 [Security issue notifications](./CONTRIBUTING.md#security-issue-notifications)
 
-## Dependencies
+## Build recipes with KMS support
 
-The only direct dependencies of this code are OpenSSL 1.0.2 or higher or 1.1.0 or higher and
-[aws-c-common](https://github.com/awslabs/aws-c-common) v0.4.42. You will also need
-a C compiler and CMake 3.9 or higher.
+We will demonstrate some simple build recipes for Linux, Mac, and Windows operating systems. These 
+recipes assume a fresh system with default installs of dependency packages.
```

Comment:
Let's define "fresh". Also s/installs/installations. 

I love the voice, really, but many of our customers are global. Let's be clear for them.